### PR TITLE
269 move bar graph below world map

### PR DESCRIPTION
**Pull request by markushal**

### CODE REVIEW

**Review by joAschauer — APPROVED** [review]
looks good to me, tested also on smaller screens.
What about the boxplots in the input data tab mentioned in #269

**Comment by markushal** [comment]
> What about the boxplots in the input data tab mentioned in #269
Oh yes, I forgot about that. I fixed this, thanks for pointing it out.

File: app/tab_input_data.py [hunk 1]
```diff
@@ -33,15 +33,10 @@ def content_input_data(api: PtxboaAPI) -> None:
             horizontal=True,
         )
 
-        # in order to keep the figures horizontally aligned, we create two st.columns
-        # pairs the columns are identified by c_{row}_{column}, zero indexed
-        c_0_0, c_0_1 = st.columns([2, 1], gap="large")
-        c_1_0, c_1_1 = st.columns([2, 1], gap="large")
-        with c_0_0:
-            st.markdown("**Map**")
+        with st.expander("**Map**"):
```
Comment (joAschauer) [inline app/tab_input_data.py:36]: I would prefer to see the map by default:
```suggestion
        with st.expander("**Map**", expanded=True):

```

**Review by joAschauer — APPROVED** [review]
Looks good! Please merge `develop` into `269-move-bar-graph-below-world-map` to resolve conflicts, then we are ready I guess.

**Pull request merged**